src/xenia/cpu/ppc/ppc_emit_memory.cc

```diff
@@ -511,6 +511,23 @@ int InstrEmit_stw(PPCHIRBuilder& f, const InstrData& i) {
   return 0;
 }
 
+int InstrEmit_stmw(PPCHIRBuilder& f, const InstrData& i) {
+  Value* b;
+
+  for (uint32_t j = 0; j < 32 - i.D.DS; ++j) {
+    if (i.D.RA == 0) {
+      b = f.LoadZeroInt64();
+    } else {
+      b = f.LoadGPR(i.D.RA + j);
+    }
+    Value* offset = f.LoadConstantInt64(XEEXTS16(i.D.DS + 4 * j));
+    f.StoreOffset(b, offset,
+                  f.ByteSwap(f.Truncate(f.LoadGPR(i.D.RA + j), INT32_TYPE)));
```

Comment:
It's RS (Register Source), not RA (Register Address). RA contains the address where to save the register to.

Reply:
It's weird that in the syntax in the documentation, RT (Register Target) is listed, but later, it's RS everywhere. But semantically RS is more correct here.

Reply:
it appears as though RS is being put in RA for these instructions.

Reply:
this appears to be correct. To verify that just the presence of these functions (ie not forcing xenia to exit immediately) was not the reason for the game getting past frame 1, I just had the functions do nothing. it crashed same as before. It appears as if the functions are actually written correctly otherwise crashing would occur.

Reply:
Maybe it's in RT actually, but will check by disassembling and matching with the values in the instruction.